CHANGES.txt

```diff
@@ -14,6 +14,8 @@ Changes
 Fixes
 -----
 
+ - Removed automatic execution of table queries.
```

Comment:
I think it is not very clear what table queries are. Maybe explain a bit more, e.g:

- Do not automatically execute the query when clicking on the "Query Table" button in the table view.